quickstarts/microsoft.netapp/anf-sap-hana/anf-sap-hana-storage/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,271 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "virtualMachines_avgvm_name": {
+            "defaultValue": "avgvm",
+            "type": "String"
+        },
+        "virtualMachines_vmSize": {
+            "defaultValue": "Standard_M8ms",
+            "type": "String",
+            "allowedValues": [
+                "Standard_M8ms"
+            ]
+        },
+        "virtualNetworks_vnet1_name": {
+            "defaultValue": "vnet1",
+            "type": "String"
+        },
+        "netAppAccounts_ANFaccount_name": {
+            "defaultValue": "ANFaccount",
+            "type": "String"
+        },
+        "networkInterfaces_avgvm980_name": {
+            "defaultValue": "avgvm980",
+            "type": "String"
+        },
+        "proximityPlacementGroups_avgppg_name": {
+            "defaultValue": "avgppg",
+            "type": "String"
+        },
+        "adminUsername": {
+            "type": "String"
+        },
+        "adminPassword": {
+            "type": "SecureString"
+        },
+        "location": {
+             "type": "String",
+            "defaultValue": "[resourceGroup().location]"
+        },
+        "availabilitySets_name": {
+            "type": "String",
+            "defaultValue": "avgavset"
+        },
+        "deployVM": {
+            "type": "bool",
+            "defaultValue": "false"
```

Comment:
this must be a bool, i.e. remove the quotes

```"defaultValue": false```

Reply:
done